packages/forms-web-app/src/dynamic-forms/dynamic-components/utils/question-utils.js

```diff
@@ -42,3 +42,10 @@ exports.getListedBuildingForQuestion = (journeyResponse, fieldName) => {
 
 	return listedBuildings.filter((listedBuilding) => listedBuilding.fieldName == fieldName);
 };
+
+/** @type {(maybeDivider: import('../../question-props').Option) => maybeDivider is { divider: string }} */
+exports.optionIsDivider = (maybeDivider) => Object.hasOwn(maybeDivider, 'divider');
+
+/** @type {(conditional: unknown) => conditional is { html: string }} */
+exports.conditionalIsJustHTML = (conditional) =>
+	!!conditional && Object.hasOwn(conditional, 'html') && Object.keys(conditional).length === 1;
```

Comment:
when does this happen?

Reply:
This is a bit of an annoying one. I don't _think_ this check will ever be true. It's not until a few lines after this that the conditional value might be overwritten with compiled nunjucks.
```
			// handle conditional (dependant) fields & set their answers
			if (optionData.conditional !== undefined) {
				if (conditionalIsJustHTML(optionData.conditional)) continue;

				let conditionalField = { ...optionData.conditional };

				conditionalField.fieldName = getConditionalFieldName(
					this.fieldName,
					conditionalField.fieldName
				);
				conditionalField.value = payload
					? payload[conditionalField.fieldName]
					: journey.response.answers[conditionalField.fieldName] || '';

				optionData.conditional = {
					html: nunjucks.render(`./dynamic-components/conditional/${conditionalField.type}.njk`, {
						payload,
						...conditionalField,
						...customViewData
					})
				};
			}
```
I'd love to make this work in a smoother way at some point but I tried to alter the internals as little as possible to avoid regressions in this.